doc/api_assets/api.js

```diff
@@ -2,24 +2,25 @@
 
 {
   function setupTheme() {
-    const kCustomPreference = 'customDarkTheme';
-    const userSettings = sessionStorage.getItem(kCustomPreference);
+    const storedTheme = localStorage.getItem('theme');
     const themeToggleButton = document.getElementById('theme-toggle-btn');
 
-    if (userSettings === null && window.matchMedia) {
+    // Follow operating system theme preference
+    if (storedTheme === null && window.matchMedia) {
       const mq = window.matchMedia('(prefers-color-scheme: dark)');
 
       if ('onchange' in mq) {
+
```

Comment:
Are these empty line changes necessary? It just adds unnecessary noise here

Reply:
I would say it is a matter of taste :). 
Before this part of code seemed pretty dense to me 🤷🏻‍♂️

Reply:
This is a personal opinion and unrelated to the code. Note that this code is all going to be thrown away in the near future, so does it really matter? 😅